packages/devtools_app/test/app_size_screen_test.dart

```diff
@@ -451,9 +451,9 @@ class AppSizeTestController extends AppSizeController {
 
   @override
   void loadDiffTreeFromJsonFiles({
-    @required DevToolsJsonFile oldFile,
-    @required DevToolsJsonFile newFile,
-    @required void Function(String error) onError,
+    required DevToolsJsonFile? oldFile,
+    required DevToolsJsonFile? newFile,
```

Comment:
make non nullable on the test controller and the super class

Reply:
postponed the file